components/utils.py

```diff
@@ -342,6 +357,27 @@ def move(source, destination):
     sudo(['mv', source, destination])
 
 
+def set_permissions_owner_execute_only(path):
+    os.chmod(
+        path,
+        # Set read and execute for owner
+        stat.S_IRUSR | stat.S_IXUSR,
+    )
+
+
+def set_ownership_recursive(user, path, group=''):
```

Comment:
please use this: https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/228/files#diff-23cb882db3b8314e49ad99212e1c4118R441